games-emulation/jrommanager/jrommanager-1.6-r1.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN=JRomManager
+MY_PV=1.6b20
+MY_P="${MY_PN}-${MY_PV}"
+
+inherit desktop
+
+DESCRIPTION="A Mame and Retrogaming Rom Manager, Java alternative to ClrMamePro"
+HOMEPAGE="https://github.com/optyfr/JRomManager"
+SRC_URI="https://github.com/optyfr/${MY_PN}/releases/download/${MY_PV}/${MY_P}.zip -> ${P}.zip"
+QA_PREBUILT="*"
+
+S="${WORKDIR}"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND=">=virtual/jre-1.8"
+DEPEND="${DEPEND}
+	app-arch/unzip"
+
+src_prepare() {
+	default
+
+	sed -e "/dir=.*/s:.*:cd ${EROOT}/usr/share/${PN}:" -i *.sh || die "sed failed!"
```

Comment:
EROOT can contain spaces, so you need to quote it. Also, you want `|| exit` inside that script since otherwise it will continue trying to do its thing even if the directory doesn't exist.

Reply:
OK will do it